topics/scapy/index.md

```diff
@@ -0,0 +1,15 @@
+---
+aliases: scapy
+created_by: Philippe Biondi
+display_name: scapy
+github_url: https://github.com/secdev/scapy
+logo: scapy.png
+released: April 19, 2021
+short_description: Scapy is a powerful interactive packet manipulation program.
```

Comment:
Could you use:
`Scapy is a Python-based interactive packet manipulation program & library. It upports Python 2 & Python 3.`